calico_containers/tests/st/docker_host.py

```diff
@@ -15,16 +15,21 @@ def __init__(self, name):
         pwd = sh.pwd().stdout.rstrip()
         docker.run("--privileged", "-v", pwd+":/code", "--name", self.name, "-tid", "jpetazzo/dind")
```

Comment:
Yeah, it kinda feels that way to me too. It should be noted though that we aren't using the host containers primarily for isolation, but instead for rapid testing of calico on multiple hosts.

Seems like your suggestion is the direction that we should move in, but also that at this point I should probably start doing something other than improving the test framework now that we can easily write multi-host tests in python.